RadixWallet/Features/GatewaySettingsFeature/Components/GatewayRow/GatewayRow+View.swift

```diff
@@ -46,33 +29,33 @@ extension GatewayRow {
 		}
 
 		public var body: some SwiftUI.View {
-			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+			WithPerceptionTracking {
 				Button(
 					action: {
-						viewStore.send(.didSelect)
+						store.send(.view(.didSelect))
 					}, label: {
-						HStack {
-							Image(asset: AssetResource.checkmarkBig)
+						HStack(spacing: .zero) {
+							Image(asset: AssetResource.check)
```

Comment:
We should be building a custom `HStack` here, but instead using our common `PlainListRow`. This way we can be sure that we have the same UI implementation everywhere we use these kind of rows. 

For this case, you probably want to use a `PlainListRowCore` with `title` and `detail` set, and have deletion button as the `Accesory` while the check image as the `Icon`.